x-pack/test/functional/page_objects/search_api_keys.ts

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
```

Comment:
these tests are going to collide with Yan's changes. They are broadly similar but theres likely slight differences in approach.

I suggest we wait until @yansavitski changes are merged in then we can rebase the FTR tests.